vowpalwabbit/parse_args.cc

```diff
@@ -445,8 +445,8 @@ input_options parse_source(vw& all, options_i& options)
       .add(make_option("no_stdin", all.stdin_off).help("do not default to reading from stdin"))
       .add(make_option("no_daemon", all.no_daemon).help("Force a loaded daemon or active learning model to accept local input instead of starting in daemon mode"))
       .add(make_option("chain_hash", parsed_options.chain_hash)
-               .help("enable chain hash for feature name and string feature value. e.g. {'A': {'B': 'C'}} is hashed as A^B^C"));
-
+               .help("enable chain hash for feature name and string feature value. e.g. {'A': {'B': 'C'}} is hashed as A^B^C"))
+      .add(make_option("flatbuffer", parsed_options.flatbuffer).help("Input a FlatBuffer File"));
```

Comment:
Suggestion: change description to "data file will be interpreted as a flatbuffer file"